compendium/ServiceComponent/include/cppmicroservices/servicecomponent/detail/ComponentInstanceImpl.hpp

```diff
@@ -417,10 +417,15 @@ class ComponentInstanceImpl final
   {
     static_assert(
       std::is_default_constructible<C>::value,
-      "Default Constructor or Constructor with AnyMap input parameter "
-      "expected when injection is false. This error happens when an "
-      "appropriate constructor is not available or when the class being "
-      "validated is not a complete type.");
+      "An appropriate constructor was not found for the Service Instance. "
+      "The manifest.json inject-references property is false so the build "
+      "was looking for a default constructor or a constructor with an "
+      "AnyMap input parameter to receive the configuration properties. "
+      "All methods for the interfaces the Service Instance is implementing "
+      "must be present. Please see "
+      "https://confluence.mathworks.com/display/CppMicroServices/"
+      "Build+Failure+When+Constructing+Service+Instance "
```

Comment:
this is a MW internal website. We can't have this in the output.

Reply:
I think the content of the page is okay? It doesn't have anything MW specific. I just need to move it someplace else. Do you have a recommendation?

Reply:
Do we really need a URL that describes the problem in the error message? I suggest removing the URL from the error message.